workflows/sarek.nf

```diff
@@ -1308,6 +1308,13 @@ def flowcellLaneFromFastq(path) {
     return fcid
 }
 
+// Check the parameters tools or skip_tools, then compare it against the provided tool
+// Returns true/false based on whether 'tool' is found in 'parameter'
+def checkTools(parameter, tool) {
```

Comment:
it seems like the function `checkTools` could also be used for other parameters which don't have anything to do with tools (like say `params.aligner` or `params.step`), so perhaps the function might as well be called something more general like `contains`(If we and nextflow can handle that kind of "name" overloading.)

Anyways, it's just a thought, and we can also just go with `checkTools` for now, and see which way the wind blows.